scripts/semver-check.sh

```diff
@@ -16,8 +16,9 @@ TAGGED_RELEASE_VERSION=$1
 LAST_VERSION=$2
 API_COMPAT_LEVEL=$3
 
+echo "Semver-check TAGGED_RELEASE_VERSION:$TAGGED_RELEASE_VERSION, LAST_VERSION: $LAST_VERSION, API_COMPAT_LEVEL: $API_COMPAT_LEVEL"
 # Checking if tagged release has API breaks
-if [[ ! -z ${TAGGED_RELEASE_VERSION} ]] && [[ ! ${TAGGED_RELEASE_VERSION} =~ "rc." ]] && [[ ! ${LAST_VERSION} =~ "rc." ]]; then
+if [[ ! -z ${TAGGED_RELEASE_VERSION} ]]; then
```

Comment:
are we currently only checking for release version? don't we check API breaks between beta and rc?

Reply:
We check all the tagged releases. The current logic doesn't exclude beta and rc.

Reply:
would it be better to know if there's a breaking change during pre-release phase?

Reply:
oh, sorry I missed that it would trigger for all the release tags. no issue for me then :)